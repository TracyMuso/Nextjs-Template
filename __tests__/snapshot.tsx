/**
 * @jest-environment jsdom
 */
import { render } from '@testing-library/react'
import Home from '../app/page'

it('renders greeting 3', () => {
  const { container } = render(<Home />)
  expect(container).toMatchSnapshot()
})
