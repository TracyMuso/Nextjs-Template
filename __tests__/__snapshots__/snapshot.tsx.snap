// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders greeting 3`] = `
<div>
  <div>
    <main>
      <h1>
        hello
      </h1>
    </main>
  </div>
</div>
`;

exports[`renders greeting 3 1`] = `
<div>
  <div>
    <main>
      <h1>
        hello
      </h1>
    </main>
  </div>
</div>
`;
